.pre-commit-config.yaml

```
@@ -1619,6 +1619,21 @@ repos:
         types: [python]
         files: ^providers/.*/src/airflow/providers/.*\.py$
         require_serial: true
+      - id: check-airflow-version-checks-in-core
+        language: pygrep
+        name: No AIRFLOW_V_3* imports in airflow-core
+        entry: "import AIRFLOW_V_3"
+        files: ^airflow-core/.*\.py$
+        pass_filenames: true
+        exclude: >
+          (?x)
+           # TODO (@amoghrajesh): revisit last few in this list as they all rely on versioned secrets masker imports
```

A bit odd to have the comment in the regex :) 
```suggestion
         # TODO (@amoghrajesh): revisit last few in this list as they all rely on versioned secrets masker imports
        exclude: >
          (?x)
```

---

Oh my bad lol